src/main/java/core/basesyntax/SkillOfCalculation.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface SkillOfCalculation {

```

Comment:
Interface should be named as described in [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-verbs-for-classinterface-names). Or you can check [this guide](https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names)